.github/actions/has-changesets/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Has Changesets'
```

Comment:
This is a "composite action", or reusable action. No secrets are needed or used here. 

Fyi, "action" is a single step in a job, "jobs" consist of multiple steps, and "workflow" consists of multiple jobs. 

https://docs.github.com/en/actions/creating-actions/creating-a-composite-action